### ISSUE TITLE
Delayed coverage loading when inspecting a card

### ISSUE THREAD

**Issue by TitusPosiadly** [MKT-Playbook/Hub#26]
### Discussed in https://github.com/MKT-Playbook/Hub/discussions/16

<div type='discussions-op-text'>

<sup>Originally posted by **TheIcyStar** November 21, 2022</sup>
Hello! Whenever I compare several d/k/gs I usually open a lot of coverage lists. However, every time it gets opened, there's a substantial delay before all of the favorite courses are visible. Of course, this is probably due to coverage calculation. But could that be done after the list is rendered without calculation, similar to the favored course list?

Basically, first, the list of courses gets rendered, then one by one the coverage-specific information gets added afterward. This way, you could see the full list of courses right away and not have to deal with an awkward scroll.

Thanks, and big ❤️ for making this awesome app!</div>
